### PR TITLE
Tidy up readme

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

**Comment by Robinlovelace** [comment]
I think the updated CLI commands are better, including how to clone abstreet for the 1st time and think this is a little better. But feedback on the CLI/bash use v. welcome, may be able to improve this to further boost reproducibility and ease of use.

**Comment by natesheehan** [comment]
Agreed this looks much tidier to me! Thanks Robin 🎉

File: README.md [hunk 1]
```diff
@@ -139,13 +135,13 @@ schema](https://a-b-street.github.io/docs/tech/dev/formats/scenarios.html).
 
 ``` r
 output_json = ab_json(output_sf, time_fun = ab_time_normal, scenario_name = "Montlake Example")
-ab_save(output_json, f = "montlake_scenarios.json")
+ab_save(output_json, f = "montlake.json")
```
Comment (natesheehan) [inline README.md:138]: 👍

File: README.md [hunk 2]
```diff
@@ -218,17 +214,21 @@ a command.
...
-2.  From the main A/B Street directory, run the following command:
+2.  From the main A/B Street repo directory, run the following commands
+    (commented lines of code clone and set the working directory,
+    replace `../montlake.json` with the path to the scenario file:
 
 ``` bash
-./cli import-scenario --input path/to/scenario.json --map data/system/us/seattle/maps/montlake.bin
+# git clone git@github.com:a-b-street/abstreet
+# cd a-b-street
+./cli import-scenario --input ../montlake.json --map data/system/us/seattle/maps/montlake.bin
```
Comment (natesheehan) [inline README.md:224]: Nice to document the full process 👍

**Review by natesheehan — APPROVED** [review]

**Pull request merged**